data/migrations/script.py.mako

```diff
@@ -0,0 +1,24 @@
+"""${message}
```

Comment:
I read your README multiple times and didn't see anything related to script.py.maki, what is the purpose of this?

Reply:
It is an autogenerated file used if you want to run alembic manually (instead of by code as I do right now). I don't delete it just in case we need to do some remediation manually but I don't see a reason to include it in the README as it is a corner case, do you want me to add it? In the general README or the migration README?